andela/wger_vulcan
# Ft present deactivated users 142465233

#### What does this PR do? 
Adds status - active or inactive - to the list of gym members and administrators.

#### Description of Task to be completed? 
At the moment, the only way to determine whether a gym member is active or not is to open his detail view. A better alternative would be to list only active members in the default list and have a second overview page for the deactivated ones.

#### How should this be manually tested? 
In development mode, navigate to the root folder of the application using ypur terminal and run the command:

`python manage.py runserver`

Open the development server address created `http://127.0.0.1:8000` , log in as admin and pick a gym. The status of members, trainers and administrators should show alongside other information. An inactive status is highlighted in red.

#### Any background context you want to provide? 
None

#### What are the relevant pivotal tracker stories? 
142465233

#### Screenshots (if appropriate) 
![image](https://cloud.githubusercontent.com/assets/19430730/24655896/ce19ec7c-1948-11e7-97de-eed6237d6a49.png)

#### Questions:
